### ISSUE TITLE
No action '[object Object]' found

### ISSUE THREAD

**Issue by jadechip** [ArkEcosystem/core#4480]
Hi! We're running 2.7.25 in our Kubernetes cluster and have encountered the following error:
```
[2021-09-03 13:49:49.534] INFO : Starting P2P Interface
[2021-09-03 13:49:50.298] INFO : Socket worker started, PID: 97
[2021-09-03 13:49:50.305] INFO : Socket worker started, PID: 91
[2021-09-03 13:49:50.346] INFO : Setting up core-magistrate-transactions.
[2021-09-03 13:49:50.492] INFO : Starting Database Manager
[2021-09-03 13:49:50.493] INFO : Establishing Database Connection
[2021-09-03 13:49:51.750] INFO : Connecting to transaction pool
[2021-09-03 13:49:51.789] INFO : Starting Blockchain Manager :chains:
[2021-09-03 13:49:51.791] ERROR: No action '[object Object]' found
```
Any clues what the cause could be?

**Comment by KovacZan** [comment]
This will be fixed when #4477 gets merged and released.

**Comment by amacar** [comment]
Fix in #4477 will need to be ported to the main branch to :)

**Comment by jadechip** [comment]
Much appreciated gents!

**Comment by boldninja** [comment]
@jadechip new version has been released on NPM (2.7.26) which resolves this issue. Thank you for the report.
